tests/incremental/13-restart-write/09-mutex-simple-access.c

```diff
@@ -0,0 +1,23 @@
+#include <pthread.h>
```

Comment:
Maybe this test would be better in 11-restart than 13-restart-write since it's about the general restarting mechanism, not the write-only unknowns one.

Reply:
Also, I guess this is the same code and patch as 13-restart-write/01-mutex-simple, but just with different conf? Might be worth mentioning that on top of the file, so it's more obvious than manually having to compare it to all the other variations in 13-restart-write.